src/component-library/src/components.scss

```diff
@@ -0,0 +1,18 @@
+@import "../../blockquote/index";
```

Comment:
Please add a comment why this is a good temporary solution and what it will look like in the near future ☺️